utils/http/jsonrpc/server.go

```diff
@@ -101,9 +115,13 @@ func (s *Server) Start() error {
 		listener, err = net.Listen("tcp", fmt.Sprint(":", s.cfg.ServePort))
 	}
 	if err != nil {
+		fmt.Printf("Start error err %v \n", err)
 		return err
 	}
 
+	if bRegistTestAction {
```

Comment:
please don't add test code in formal logic

Reply:
ok.  remove already